### PR TITLE
New CoC and contributing guide

### PR DESCRIPTION
**Pull request by andreas131989**
Added a code of conduct doc and contributing doc.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Code of Conduct
+
+All participants to the Trow project are required to comply with the following code of conduct,
```
Comment (alex-glv) [inline CODE_OF_CONDUCT.md:3]: s/Trow/Pre-Scaling/Operator

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to adrian.mouat@container-solutions.com or any of the project admins.
```
Comment (alex-glv) [inline CODE_OF_CONDUCT.md:67]: Adrian's email, not sure if that needs to change

**Review by alex-glv — CHANGES_REQUESTED** [review]
A few small changes otherwise 👍

**Review by amouat — APPROVED** [review]
LGTM :+1: 

There are some changes to the README that could arguably be a different PR but I don't think it's worth splitting them out.

**Pull request merged**